app/lib/provider/network/server/controller/receive_controller.dart

```diff
@@ -441,17 +444,15 @@ class ReceiveController {
         ),
       ),
     );
+    final fileType = receivingFile.file.fileType;
+    final saveToGallery = receiveState.saveToGallery && (fileType == FileType.image || fileType == FileType.video);
 
+    final (destinationPath, documentUri, finalName) = await digestFilePathAndPrepareDirectory(
```

Comment:
Why move this outside of try-catch?

Reply:
i wanted to access the the destinationPath variable when file is saved and we navigate to homepage . the `digestFilePathAndPrepareDirectory()` did't have any exception signature so i put it outisde the try cactch